R/redist_map.R

```diff
@@ -9,8 +9,10 @@
 # constructors and reconstructors
 
 # Main internal constructor
-new_redist_map <- function(data, adj, ndists, pop_bounds, pop_col = "pop",
-                           adj_col = "adj", add_adj = TRUE, existing_col = NULL) {
+new_redist_map <- function(
+        data, adj, ndists, total_seats, district_seat_sizes,
```

Comment:
note for @christopherkenny : this is a breaking change in arg order but I'm actually OK with it because it's weird to add additional seats args nowhere near `ndists`